cypress/integration/profile/update-tls.spec.ts

```diff
@@ -0,0 +1,178 @@
+/*********************************************************************
```

Comment:
I think lets rename this file to "update.spec.ts" since it tests other updates besides TLS (wireless config, network configuration static) -- its more generic.

Reply:
I agree. I thought updating wireless config, network configurations are one-off cases so I included them in the same file. If you think it will be easier to read (maybe extend coverage) in future then I can split into multiple spec file? Or I can just rename the file as you suggested.